fairworkflows/fairstep.py

```diff
@@ -381,6 +389,8 @@ def is_fairstep(label: str = None, is_pplan_step: bool = True, is_manual_task: b
         is_manual_task (str): Denotes whether this step is a bpmn.ManualTask
         is_script_task (str): Denotes whether this step is a bpmn.ScriptTask
 
+    All additional arguments are expected to correspond to input parameters of the decorated
```

Comment:
I think we should add a bit more documentation here. Especially on declaring semantic types for the output arguments.

Reply:
Yes, that's true, the documentation is really lacking here. I have added more to the docstring, including an example.